english/dream/tender-defender/tender-defender.md

```diff
@@ -25,4 +25,5 @@ DO BE: ready for anything
 DON'T BE: completely oblivious or totally anxious
 
 If you can do all these things, you will be on your way to becoming 
-[what women want](http://google.com)
\ No newline at end of file
+[what women want](http://google.com)
+[this is the correct link:](http://www.goodreads.com/book/show/24396873-mate)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.